sources/Valkey.Glide/Abstract/ConfigurationOptions.cs

```diff
@@ -476,9 +486,80 @@ private ConfigurationOptions DoParse(string configuration, bool ignoreUnknown =
                 }
             }
         }
+
+        // Validate ReadFrom configuration after all parameters have been parsed
+        if (tempReadFromStrategy.HasValue)
+        {
+            ReadFrom = SetReadFrom(tempReadFromStrategy, tempAz);
+        }
+
         return this;
     }
 
+    private string ParseAzParameter(string value)
```

Comment:
This really isn't parsing it and in fact, just checks if it's null/empty. It possibly doesn't need to be a helper.